app/models/ability.rb

```diff
@@ -112,6 +131,6 @@ def curation_concerns
     end
 
     def roles
-      ['anonymous', 'campus_patron', 'curator', 'fulfiller', 'editor', 'image_editor', 'admin']
+      ['anonymous', 'music_patron', 'campus_patron', 'curator', 'fulfiller', 'editor', 'image_editor', 'admin']
```

Comment:
Move this list (or, at least, the parts not those which must always be present) to configuration?